### ISSUE TITLE
Save entity information in Box of Eternal Closure

### ISSUE THREAD

**Issue by Paril** [rubensworks/EvilCraft#653]
#### Issue type: 🐛  ➕ 

____
#### Short description:
Charged Creepers don't retain their chargedness in the box, so you can't duplicate any drops that might be special to them (Reliquary, etc).

#### Expected behaviour:
Box contains Charged Creeper

#### Actual behaviour:
Box contains regular Creeper

#### Steps to reproduce the problem:

1. Get Charged Creeper
2. Kill
3. Box

____
#### Versions:

_Exact versions of the following mods, not just *latest*._

- This mod: EvilCraft-1.12.2-0.10.44.jar
- CyclopsCore: CyclopsCore-1.12.2-0.11.1.jar
- Minecraft: 1.12
- Forge: forge-1.12.2-14.23.1.2590-universal.jar

**Comment by rubensworks** [comment]
Yep, that's because the boxes _only_ store entity type at the moment. No entity NBT information is stored at the moment.
